### PR TITLE
Add box plot and exert of composition data

### PR DESCRIPTION
**Pull request by Pattrigue**
Added an exert of the composition data (in a table) and a box plot describing it. Also added some additional text describing the composition data.

### CODE REVIEW

File: report_pre_thesis/src/sections/data_overview.tex [hunk 1]
```diff
@@ -98,6 +98,38 @@ \section{Data Overview}\label{sec:data_overview}
 	\label{fig:masked_regions}
 \end{figure}
 
+\subsection{Composition Data}\label{subsec:composition_data}
 In addition to these datasets, there is also a \\ \texttt{ccam\_calibration\_compositions.csv} file that contains ground truth data for each major oxide in each sample.
 There are a total of eight major oxides: \ce{SiO2}, \ce{TiO2}, \ce{Al2O3}, \ce{FeOT}, \ce{MnO}, \ce{MgO}, \ce{CaO}, \ce{Na2O}, and \ce{K2O}.
-For each of these oxides, the data specifies their respective concentrations in each sample, expressed as a weight percentage (wt. \%) of the total composition.
\ No newline at end of file
+For each of these oxides, the data specifies their respective concentrations in each sample, expressed as a weight percentage (wt. \%) of the total composition.
+An exert of this dataset is shown in Table~\ref{tab:composotion_data_example}.
```
Comment (Ivikhostrup) [inline report_pre_thesis/src/sections/data_overview.tex:105]:
```suggestion
An excerpt of this dataset is shown in Table~\ref{tab:composotion_data_example}.
```

File: report_pre_thesis/src/sections/data_overview.tex [hunk 2]
```diff
@@ -98,6 +98,38 @@ \section{Data Overview}\label{sec:data_overview}
...
+
+\begin{figure*}
+	\centering
+	\includegraphics[width=0.85\textwidth]{images/composition_box_plot.png}
+	\caption{Box plot of the composition data. The orange line represents the median, the black boxes represent the interquartile range for each oxide, and the whiskers represent the range of the data. The black circles represent outliers.}
+	\label{fig:composition_box_plot}
+\end{figure*}
+
+Figure \ref{fig:composition_box_plot} shows a box plot of the composition data.
+As can be seen in the figure, the data contains several outliers, which are samples that are significantly different from the rest of the data.
```
Comment (Ivikhostrup) [inline report_pre_thesis/src/sections/data_overview.tex:132]:
```suggestion
...which are samples that are significantly different from the rest of the data.
```
Isnt this the defintion of outliers? Dont know if you need this explanation

Comment (Ivikhostrup) [inline report_pre_thesis/src/sections/data_overview.tex:132]:
```
...which are samples that are significantly different from the rest of the data.
```
Isnt this the definition of outliers? Not sure you need to include this

Comment (chhoumann) [reply report_pre_thesis/src/sections/data_overview.tex:132]: Depends on how you define outliers

File: report_pre_thesis/src/sections/data_overview.tex [hunk 3]
```diff
@@ -98,6 +98,37 @@ \section{Data Overview}\label{sec:data_overview}
...
```
Comment (Pattrigue) [inline report_pre_thesis/src/sections/data_overview.tex:132]:
```suggestion
```

**Pull request merged**